emulator_test.go

```diff
@@ -59,6 +60,8 @@ func createEmulator(samplingRate int, phaseOffsetDeg float64) *Emulator {
 		NoiseMax:                        0.01,
 		InstantaneousAnomalyMagnitude:   30,
 		InstantaneousAnomalyProbability: 0.01,
+		TrendAnomalyTimeStart:           time.Now().Unix(),
```

Comment:
The problem with defining `TrendAnomalyTimeStart` at object initialisation is it will be dependent on when the object is created, and only iterate through the anomaly once (I think?).

Also, we probably want to avoid doing this in "real time" (i.e. we're incrementing time with each call to `.Step()` which is not limited to wall time)